### PR TITLE
Added German translation file

### PR DESCRIPTION
**Pull request by AndreasWebdev**
I am not entirely sure how to modify the other scripts to enable German language support, but here are all strings translated.

### CODE REVIEW

**Comment by samuelmaddock** [comment]
I'll hopefully get to this sometime this week. I need to prioritize a few other things first. 👍

File: app/locale/de_DE.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
+export default {
+  add: 'Hinzufügen',
+  addMedia: 'Medien hinzufügen',
+  advanced: 'Erweitert',
+  allowTracking: 'App Informationen senden',
```
Comment (nicmr) [inline app/locale/de_DE.ts]: allowTracking: 'Nutzungsdaten senden'

File: app/locale/de_DE.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  allowTrackingDesc:
+    'Sende anonyme Informationsstatistiken an den Entwickler um diese App besser zu machen.',
+  cancel: 'Abbrechen',
+  chatColor: 'Chat Farbe',
```
Comment (nicmr) [inline app/locale/de_DE.ts]: chatColor: 'Chatfarbe'

File: app/locale/de_DE.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  chooseUsername: 'Wähle deinen Benutzernamen',
+  connecting: 'Verbinden',
+  continue: 'Weiter',
+  copy: 'Kopieren',
+  copyLink: 'Link kopieren',
+  debug: 'Debug',
+  developerMode: 'Entwicklermodus',
+  developerModeDesc: 'Zeige Entwicklerwerkzeuge um Webseiten zu untersuchen.',
+  disconnect: 'Verbindung trennen',
+  duplicate: 'Duplikat',
```
Comment (nicmr) [inline app/locale/de_DE.ts]: duplicate: 'Duplizieren'

**Comment by AndreasWebdev** [comment]
Did Github automatically add the changes from @nicmr ?

**Comment by samuelmaddock** [comment]
I'm going to merge in these changes, but there's actually no support for localization yet. You can follow #5 for progress.

**Pull request merged**